perf_test

```diff
@@ -29,17 +29,19 @@ def iops_suffix(val):
     suffixes = ["", "K", "M", "G"]
     return suffix(val, suffixes, base=1000)
 
-def run_fio(fio, bs, chunk_size, thread_cnt, cache_size, bw, write):
+def run_fio(fio, bs, chunk_size, thread_cnt, cache_size, bw, write, stripe_size):
     res = {"blocksize"   : bs,
            "chunk_size"  : chunk_size,
            "thread_cnt"  : thread_cnt,
            "cache_size"  : cache_size,
     }
 
     rw = "write" if write else "read"
+    offset_increment = fio.args["size"] + stripe_size
```

Comment:
FIO args accepts suffix strings, need a conversion function here

Reply:
this also doesn't align the offset_increment to the stripe_size, should be (size + stripe_size - 1) // stripe_size * stripe_size

Reply:
> FIO args accepts suffix strings, need a conversion function here

the `size` is a hardcoded constant (for now at least). And `stripe_size` is computed from `chunk_size`, which is already converted [during the arg parsing](https://github.com/Eideticom/raid5-tests/blob/cecded61d38276c0d08b66b2c286994ac25318b0/perf_test#L99).

> this also doesn't align the offset_increment to the stripe_size, should be (size + stripe_size - 1) // stripe_size * stripe_size

Hmm right, will fix

Reply:
But they are a set of args passed directly into fio, I'm assuming this is based off of the noloadfs one and adding an fio arg is as simple as adding an FIO arg.  The expectation should be fio's expectation so we don't need to keep track of what's special and what's just an fio arg.

Reply:
Yeah, this is based on the noloadfs fio.py. But note that this offset_increment computation is done in the perf_test script, not on fio.py, and in the perf_test it's not possible to change the `size` argument.

Although I supposed if we changed the base fio.py then this would break... So, sure, I'll add a suffix to bytes conversion

Reply:
Yeah, this is definitely confusing but I think the problem is with the original code. We have self.args which looks to almost entirely be a compile time constant even though it is created in __init__.... then we add self.SIZE which is a compile time constant and repeated in self.args; -- also a constant.

And thus it's not clear how to treat args at all.... David claims we have to impose fios rules on it, though why that's the case isn't clear. Worse yet we have something called args that doesn't seem like a sensible inter-module interface being used from another module.... 

Just because you have a variable with no underscores and nobody indicated it was private doesn't mean it's a good idea to just treat it as public interface. That's how you create a giant ball of spaghetti. Nobody reading the Fio code would think that args["size"] is a public interface with format promises to some other module.

If the calling code cares about the size, then it should probably be passing that size to the Fio class. Not having Fio specify it as a constant in a poorly named array and then have the calling code grab it from the class's variables.